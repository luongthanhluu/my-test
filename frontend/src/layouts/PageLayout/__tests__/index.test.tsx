import React from 'react'
import { render } from 'enzyme'

import { PageLayout } from '../index'

test('Loading with false props', () => {
    const wrapper = render(
        <PageLayout title="title">
            <div></div>
        </PageLayout>
    )
    expect(wrapper).toMatchSnapshot()
})
