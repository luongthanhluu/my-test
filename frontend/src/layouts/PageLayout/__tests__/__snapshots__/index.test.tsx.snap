// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading with false props 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
    >
      <div
        class="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
      >
        title
      </div>
    </div>
  </header>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-13kvln1-MuiContainer-root"
  >
    <div />
  </div>
</div>
`;
